go/throttle/throttler.go

```diff
@@ -250,6 +256,16 @@ func (throttler *Throttler) aggregateMySQLMetrics() error {
 		ignoreHostsCount := throttler.mysqlInventory.IgnoreHostsCount[clusterName]
 		aggregatedMetric := aggregateMySQLProbes(probes, throttler.mysqlInventory.InstanceKeyMetrics, ignoreHostsCount)
 		go throttler.aggregatedMetrics.Set(metricName, aggregatedMetric, cache.DefaultExpiration)
+		if throttler.memcacheClient != nil {
+			go func() {
+				value, err := aggregatedMetric.Get()
+				if err != nil {
+					throttler.memcacheClient.Set(&memcache.Item{Key: metricName, Value: []byte("")})
```

Comment:
How long will these metrics leave? 

I think we should provide the shortest TTL Memcache allows (1 second), and also (maybe this is not necessary) store the epoch with millisecond resolution at which the value has been cached. Otherwise, if by any reason the cache is not populated (f.i. a config change made freno to stop reporting to memcache) we could be reading stale data.